src/Tbody.js

```diff
@@ -1,15 +1,10 @@
 import React from "react";
-import fakeBookings from "./fakeBookings.js";
 import moment from "moment";
-const Tbody = () => {
+const Tbody = props => {
+  console.log(props);
```

Comment:
Did you attempt exercise 16? I think you'll need to update this component a bit to be able to do that exercise.